core/math/aabb.h

```diff
@@ -122,6 +122,39 @@ struct _NO_DISCARD_ AABB {
 		return position + (size * 0.5f);
 	}
 
+	_FORCE_INLINE_ AABB lerp(const AABB &p_to, const real_t p_weight) const {
+		AABB res = *this;
+		res.position.x = Math::lerp(res.position.x, p_to.position.x, p_weight);
+		res.position.y = Math::lerp(res.position.y, p_to.position.y, p_weight);
+		res.position.z = Math::lerp(res.position.z, p_to.position.z, p_weight);
+		res.size.x = Math::lerp(res.size.x, p_to.size.x, p_weight);
+		res.size.y = Math::lerp(res.size.y, p_to.size.y, p_weight);
+		res.size.z = Math::lerp(res.size.z, p_to.size.z, p_weight);
+		return res;
```

Comment:
```suggestion
		res.size.x = Math::abs(Math::lerp(res.size.x, p_to.size.x, p_weight));
		res.size.y = Math::abs(Math::lerp(res.size.y, p_to.size.y, p_weight));
		res.size.z = Math::abs(Math::lerp(res.size.z, p_to.size.z, p_weight));
		return res;
```
Needs to apply for other parts.

Reply:
@TokageItLab should I apply the `Math::abs(Math::...` to all propreties present in the methods `lerp`, `cubic_interpolate`, `cubic_interpolate_in_time`? In case of `AABB` class it would be applied to `size` and `position` and in `Rect2` class, it would be applied to `position`, `width` and `height` props.

Reply:
Only needs to be applied for size; negative values are allowed for position, but should not be for size.

Reply:
@TokageItLab done!